components/get_link_modal.tsx

```diff
@@ -65,6 +66,7 @@ export default class GetLinkModal extends React.PureComponent<Props, State> {
         }
 
         let copyLink = null;
+        const linkAvailable = this.props.link;
```

Comment:
Can we make this boolean explicit?
```suggestion
         const linkAvailable = Boolean(this.props.link);
```

Reply:
Added in the latest commit.